src/features/list-view/ListView.js

```diff
@@ -2,10 +2,13 @@
 import * as React from 'react';
 import classNames from 'classnames';
 import noop from 'lodash/noop';
+import uniqueId from 'lodash/uniqueId';
+// import { FormattedMessage } from 'react-intl';
```

Comment:
Will uncomment once the relevant code from the parent app is ported over to BUI.